doc/src/manual/control-flow.md

```diff
@@ -440,7 +442,26 @@ julia> j
 ERROR: UndefVarError: j not defined
 ```
 
-See [Scope of Variables](@ref scope-of-variables) for a detailed explanation of variable scope and how it works in
+```jldoctest
+julia> j = 0
+0
+
+julia> for j = 1:5
```

Comment:
No need to go up all the way to `5` IMO. `3` is perhaps enough.

Reply:
Yes. I'll change all `1:5` examples in that section to `1:3` to be consistent.